src/koopman/dmd_algorithms.jl

```diff
@@ -0,0 +1,6 @@
+struct DMDPINV <: abstractDMDAlg end;
+struct DMDSVD <: abstractDMDAlg end;
+
+function estimate_operator(alg::DMDPINV, X, Y; atol::Real = 0, rtol::Real = 0)
+    return Y * pinv(X; atol = atol, rtol = rtol)
```

Comment:
Can't this be done with an svd(X)/Y ?

Reply:
Could be done with `(svd(X')\Y')'` but right now, I do not see the advantage. Doesn't pinv use the svd? And it runs faster for me. I will ues the svd however for a different algorithm.